core/metadata.js

```diff
@@ -814,6 +816,13 @@ function shouldIgnore(
   })
 }
 
-function updateLocal(doc /*: Metadata */, newLocal /*: Object */) {
-  doc.local = _.pick(newLocal || doc, LOCAL_ATTRIBUTES)
+function updateLocal(doc /*: Metadata */, newLocal /*: ?Object */ = {}) {
+  // Boolean attributes not set in doc when false will not override an existing
+  // truthy value.
+  // This is the case for `executable` and we need to provide a default falsy
+  // value to override the `newLocal` executable value in all cases.
+  doc.local = _.pick(
+    _.defaults(newLocal, { executable: false }, doc),
+    LOCAL_ATTRIBUTES
+  )
```

Comment:
Is this change related to this PR? It looks like it is more a bugfix that may be in a separated commit.

Reply:
The use of `_.defaults()` is needed to set the `updated_at` value here : https://github.com/cozy-labs/cozy-desktop/pull/1661/files#diff-b2fbff412cdb1b20ee3b7c797ef5b609R772.

But using this method means the `executable: true` attribute from `doc` can override the absence of the `executable` attribute in `newLocal` so we need to provide the intermediary default `executable:false` value.
So it is related to this PR.